src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE_FORMAT = "dd.MM.yyyy";
+    private static final String INFO_SEPARATOR = " - ";
+    private static final String DATA_SEPARATOR = " ";
+    private static final int INDEX_OF_DATA = 0;
+    private static final int INDEX_OF_NAME = 1;
+    private static final int INDEX_OF_WORK_HOURS = 2;
+    private static final int INDEX_SALARY_PER_HOUR = 3;
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = parseDateWithFormatter(dateFrom);
+        LocalDate endDate = parseDateWithFormatter(dateTo);
+        StringBuilder employeeInfo = new StringBuilder("Report for period ")
+                .append(dateFrom)
+                .append(INFO_SEPARATOR)
+                .append(dateTo);
```

Comment:
Use StringBuilder outside of loops for concatenation. Since this is outside of a loop, it's acceptable, but be mindful of using StringBuilder for concatenations inside loops.